protocols/migration/migration_helpers.py

```diff
@@ -298,29 +364,33 @@ def migrate_interpretation_request_cancer_to_latest(json_dict, assembly):
         :type assembly: Assembly
         :rtype: CancerInterpretationRequest_5_0_0
         """
-        ir_v500 = None
+        ir_v600 = None
 
-        if PayloadValidation(klass=CancerInterpretationRequest_5_0_0, payload=json_dict).is_valid:
-            ir_v500 = CancerInterpretationRequest_5_0_0.fromJsonDict(jsonDict=json_dict)
+        if PayloadValidation(klass=CancerInterpretationRequest_6_0_0, payload=json_dict).is_valid:
+            logging.info("Cancer interpretation request in models reports 6.0.0")
+            ir_v600 = CancerInterpretationRequest_6_0_0.fromJsonDict(jsonDict=json_dict)
```

Comment:
we need to migrate the `ReportVersionControl` field when moving from 5.0.0 to 6.0.0. Adding it in...